cds_migrator_kit/rdm/migration/transform/transform.py

```diff
@@ -304,21 +311,54 @@ def creator_identifiers(creator):
             inner_dict = creator.get("person_or_org", {})
             identifiers = inner_dict.get("identifiers", [])
             for identifier in identifiers:
-                # TODO process CDS and CERN Ids when names vocabulary ready
-                if identifier["scheme"] == "inspire":
+                # we check for unknown schemes
+                if identifier["scheme"] in ["inspire", "orcid", "cern", "cds"] :
                     processed_identifiers.append(identifier)
             if processed_identifiers:
-
                 inner_dict["identifiers"] = processed_identifiers
             else:
                 inner_dict.pop("identifiers", None)
 
+        def lookup_person_id(creator):
+            inner_dict = creator.get("person_or_org", {})
+            identifiers = deepcopy(inner_dict.get("identifiers", []))
+            name = None
+
+            # lookup and remove person_id
+            for identifier in reversed(identifiers):
+                if identifier.get("scheme") == "cern":
+                    name = NamesMetadata.query.filter_by(pid=identifier["identifier"]).one_or_none()
+                    # we drop CERN person id identifier
+                    identifiers.remove(identifier)
+
+            creator["person_or_org"]["identifiers"] = identifiers
+            if name:
+                # TODO decide: update name with the latest data from sync
```

Comment:
leftovers?

Reply:
to discuss actually, I wanted to run this by you guys today.